pyspark/test/bigdl/keras1/test_layer.py

```diff
@@ -55,7 +53,11 @@ def __dump_keras(self, keras_model, dump_weights=False):
         return keras_model_json_path, keras_model_hdf5_path
 
     def __generate_model(self, input_data, output_layer):
-        input_node = [Input(shape=input_data.shape[1:]) for i in input_data] # it's a list incase of Merge Layer
+        import keras
+        if isinstance(output_layer, keras.engine.Merge):  # it's a list in case of Merge Layer
+            input_node = [Input(shape=i.shape[1:]) for i in input_data]
+        else:
+            input_node = Input(shape=input_data.shape[1:])
```

Comment:
``` python
          def without_batch(batch_shape):
                   return batch_shape.shape[1:]
+        if isinstance(output_layer, keras.engine.Merge):  # it's a list in case of Merge Layer
              assert isinstance(List, input_data)
+            input = [Input(shape=without_batch(input_data.shape)) for i in input_data]
+        else:
+            input = Input(shape=without_batch(input_data.shape))
```

Reply:
??

Reply:
You mean add the following?
         ` def without_batch(batch_shape):
                   return batch_shape.shape[1:]`

Reply:
Ignored.